test/Core.Test/Utilities/CoreHelpersTests.cs

```diff
@@ -106,11 +128,26 @@ public void SecureRandomString_Success()
         }
 
         [Theory]
-        // [InlineData(long.MinValue, "8589934592 GB")]
-        // [InlineData(long.MinValue + 1, "Change")]
-        [InlineData(1000L, "1000 Bytes")]
         [InlineData(1, "1 Bytes")]
         [InlineData(-5L, "5 Bytes")]
```

Comment:
If you don't mind, we should make this method consistent by using the size variable for all the stringifications.

 I can commit the change if you don't want to deal with it.

Reply:
I agree, I would be more than happy to make that change.